src/frontend/src/components/nav/MainMenu.tsx

```diff
@@ -57,6 +67,15 @@ export function MainMenu() {
             <Trans>System Settings</Trans>
           </Menu.Item>
         )}
+        <Menu.Item
+          onClick={toggleColorScheme}
+          leftSection={colorScheme === 'dark' ? <IconSun /> : <IconMoonStars />}
+          c={
+            colorScheme === 'dark' ? vars.colors.yellow[4] : vars.colors.blue[6]
+          }
+        >
+          <Trans>Color Mode</Trans>
```

Comment:
Can the text here be a little more descriptive of the action performed, such as:

- Change Color Mode
- Toggle Color Mode

Reply:
Sure, have changed it